cmd/cluster-network-operator/k8sutil.go

```diff
@@ -72,7 +69,6 @@ func GetOperatorNamespace() (string, error) {
 		return "", err
 	}
 	ns := strings.TrimSpace(string(nsBytes))
-	log.V(1).Info("Found namespace", "Namespace", ns)
```

Comment:
You don't need to get rid of these; they have access to `var logger` in the other file.

Reply:
as Casey said, these should be kept, just using the existing logging system. (which is now `"log"` rather than "the logger in the other file")

Reply:
Done